functions.php

```diff
@@ -260,4 +260,13 @@ function getScoreboard($DB) {
     return $out;
 }
 
```

Comment:
Commenting and documenting your code is generally a good idea. If you look at every other function in `functions.php`, they have a comment above the function describing what it does